Makefile

```diff
@@ -12,14 +12,16 @@ test:
 	# Create a new project with the default values
 	cookiecutter . --no-input
 
+	virtualenv repo_name/.venv
```

Comment:
Because we run the tests of the cookiecutter'd app using that virtualenv (and I didn't want it using the virtualenv from the cookiecutter install)